cmd/dataset_tokenizer/dataset_tokenizer.go

```diff
@@ -552,18 +619,25 @@ func main() {
 		"sanitize inputs of whitespace issues")
 	reorderPaths := flag.String("reorder", "",
 		"reorder input files to specification [size_ascending, "+
-			"size_descending, name_ascending, name_descending, random, none]")
+			"size_descending, name_ascending, name_descending, random, shuffle, none]")
+	sampling := flag.Int("sampling", 100, "a integer value from 0-100 which tells the tokenizer how many chunks to discard in %, 60 keeps 60% chunks")
```

Comment:
wrap this.  You can use `"" +\ ""`.

Reply:
not super sure what you mean by that but i can just do "blahblahblah" + "blahblahblah" on the second line right?